### ISSUE TITLE
SublimeLinter is calling Flake8 with the system python instead of the Virtualenv

### ISSUE THREAD

**Issue by kevinastone** [SublimeLinter/SublimeLinter-flake8#34]
I install flake8 in a virtualenv using [pipsi](https://github.com/mitsuhiko/pipsi).  This sets up a bin directory in my path to call python scripts but keeping them isolated in virtualenvs.

It works fine when flake8 is run normally:
```
kstone@kstone-mbp ~> which flake8
$HOME/.local/bin/flake8
kstone@kstone-mbp ~> flake8 --version
2.5.0 (pep8: 1.5.7, pyflakes: 1.0.0, mccabe: 0.3.1) CPython 2.7.10 on Darwin
```

But SublimeLinter uses the system python (if another one isn't configured) rather than calling the entry point directly and thus doesn't register the appropriate virtualenv:
```
SublimeLinter: flake8 version query: /usr/bin/python $HOME/.local/bin/flake8 --version 
SublimeLinter: WARNING: no flake8 version could be extracted from:
Traceback (most recent call last):
  File "$HOME/.local/bin/flake8", line 7, in <module>
    from flake8.main import main
ImportError: No module named flake8.main
```

Basically it's calling `python ~/.local/bin/flake8` instead of just executing `~/.local/bin/flake8`

**Comment by silviot** [comment]
I had the same symptom, not fully sure if it's the same problem: on my machine the default `python` binary is 2.7, and `flake8` was not working.

I overrode the default python just for sublime by adding an executable script in `/usr/local/bin/subl` with this content:
```
#!/bin/bash
PATH=/opt/pyenv/35/bin:$PATH
exec /opt/sublime_text/sublime_text "$@"
```

If you use this solution, you need to change `/opt/pyenv/35/bin` to a path where you have a working python 3 binary.
I'm posting it here in case it's useful to others.

**Comment by kaste** [comment]
Should be fixed by #38.
